src/pretix/control/views/vouchers.py

```diff
@@ -215,7 +215,7 @@ def get_form_kwargs(self):
     @transaction.atomic
     def form_valid(self, form):
         form.instance.event = self.request.event
-        messages.success(self.request, _('The new voucher has been created.'))
+        messages.success(self.request, _('The new voucher has been created.') + ' (%s)' % form.instance.code)
```

Comment:
updated the line above.
```
messages.success(self.request, _('The new voucher has been created: {code}').format(code=form.instance.code))
```
I also updated the translation files at the same time.